Src/Enums.NET/Enums.NET.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;netcoreapp3.0;netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.1;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.1;netstandard1.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
Because of the change in TFM the item groups below should be updated for an optimal dependency tree.